src/eko/evolution_operator/__init__.py

```diff
@@ -245,7 +246,12 @@ class Operator:
             cut to the upper limit in the mellin inversion
     """
 
-    def __init__(self, config, managers, nf, q2_from, q2_to, mellin_cut=5e-2):
+    operator_type = "Evolution"
+    n_pools = int(os.cpu_count() / 2)
```

Comment:
I'm still convinced you can avoid to manually define, but it's even like this.

Reply:
okay fine, can remove the division by 2